CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## 1.3.0
+
+- Replace hash with faster and better finalized hash.
+  This replaces the previous "fxhash" algorithm originating in Firefox
+  with a custom hasher designed and implemented by Orson Peters ([`@orlp`](https://github.com/orlp)).
+  It was measured to have slightly better performance for rustc, and has better theoretical properties
+  and also includes a signficantly better string hasher.
```

Comment:
@orlp does this sound good?

Reply:
Sure. Just a grammar nitpick, `s/and has better/has better/`.